builder/common/step_run_source_vm.go

```diff
@@ -264,9 +252,9 @@ func (s *StepRunSourceVm) Run(ctx context.Context, state multistep.StateBag) mul
 		// Now tag volumes
 
 		volumeIds := make([]string, 0)
-		for _, v := range *vm.BlockDeviceMappings {
+		for _, v := range vm.BlockDeviceMappings {
 			if bsu := v.Bsu; !reflect.DeepEqual(bsu, oscgo.BsuCreated{}) {
```

Comment:
I don"t understand why the test is required (apart from the fact that reflect is unnecessary, it can be replaced by a test on VolumeID). It the VM runs, it means the volumes exist and have an ID...